dirbot/pipelines.py

```diff
@@ -10,7 +12,7 @@ class FilterWordsPipeline(object):
 
     def process_item(self, item, spider):
         for word in self.words_to_filter:
-            if word in unicode(item['description']).lower():
+            if word in six.text_type(item['description']).lower():
```

Comment:
Right. Items were being populated with lists of values. 
I changed this to single value with `.extract_first()` now, and included updated selectors for new layout.